cmd/erasure-metadata.go

```diff
@@ -323,7 +324,18 @@ func objectQuorumFromMeta(ctx context.Context, er erasureObjects, partsMetaData
 		return 0, 0, err
 	}
 
+	dataBlocks := latestFileInfo.Erasure.DataBlocks
+	parityBlocks := globalStorageClass.GetParityForSC(latestFileInfo.Metadata[xhttp.AmzStorageClass])
+	if parityBlocks == 0 {
```

Comment:
When would parity blocks be 0?

Reply:
when the ENV is missing @kerneltime or the config didn't populate the global value yet. It's a defensive code.